nixos/doc/manual/installation/installing.xml

```diff
@@ -69,9 +69,42 @@
    </para>
 
    <para>
-    To manually configure the wifi on the minimal installer, run
-    <command>wpa_supplicant -B -i interface -c &lt;(wpa_passphrase 'SSID'
-    'key')</command>.
+    To configure the wifi on the minimal installer, first start wpa_supplicant
```

Comment:
```suggestion
    If you're using the graphical installer non-graphically, use `nmtui` to configure wifi.

    On the minimal installer, first start wpa_supplicant
```

Reply:
I opted for updating the paragraph above, which is about the graphical installed.